schema_based_on_diagram.sql

```diff
@@ -0,0 +1,48 @@
+CREATE DATABASE clinic;
+

```

Comment:
- [ ] Please make sure to add indexes for each of the foreign keys (6 in total), as per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) requirement.

Reply:
Indexing added.